Modix.Data/Models/Promotions/PromotionActionCreationData.cs

```diff
@@ -26,5 +26,10 @@ public class PromotionActionCreationData
         /// See <see cref="PromotionActionEntity.CreatedById"/>.
         /// </summary>
         public ulong CreatedById { get; set; }
+
+        /// <summary>
+        ///
+        /// </summary>
+        public PromotionCampaignBrief Campaign { get; set; }
```

Comment:
Is this populated for every action? If not, it needs to be annotated as nullable.